snap/snapcraft.yaml

```diff
@@ -108,19 +108,24 @@ apps:
     adapter: full
     after:
       - postgres
+      # to ensure kong.yml is ready for kong to process as the file is set up from security-secretstore-setup
+      - security-secretstore-setup
     command: bin/kong-daemon.sh
     command-chain:
       - bin/perl5lib-launch.sh
       - bin/kong-launch.sh
     daemon: forking
     environment:
       KONG_CONF: $SNAP_DATA/config/security-proxy-setup/kong.conf
+      KONGADMIN_CONFIGFILEPATH: $SNAP_DATA/config/kong.yml
       KONG_LOGS_DIR: $SNAP_COMMON/logs
       KONG_PROXY_ACCESS_LOG: $SNAP_COMMON/logs/kong-proxy-access.log
       KONG_ADMIN_ACCESS_LOG: $SNAP_COMMON/logs/kong-admin-access.log
       KONG_PROXY_ERROR_LOG: $SNAP_COMMON/logs/kong-proxy-error.log
       KONG_ADMIN_ERROR_LOG: $SNAP_COMMON/logs/kong-admin-error.log
       KONG_ADMIN_LISTEN: "localhost:8001, localhost:8444 ssl"
+      KONG_STATUS_LISTEN: "0.0.0.0:8100"
```

Comment:
Can you listen on `127.0.0.1:8100` instead?  Snaps share the host's network stack and this exposes the port to the world.

Reply:
i think most of cases would, not sure about the corner case of "remote" configuration for consul which may need to listen not just `localhost`, correct me if I am wrong.

Reply:
Kong is our gateway, so it needs to be exposed to the world

Reply:
yea, that is what I think it should be as well

Reply:
I'm going to ask the same question as @bnevis-i, why does this port (which seems to be a read-only status API) need to be accessible from a different host? Are we actively using this status API in the new security model for the admin API?

Reply:
changed to bind to localhost only